src/hotspot/cpu/x86/continuationEntry_x86.inline.hpp

```diff
@@ -25,9 +25,8 @@
 #ifndef CPU_X86_CONTINUATIONENTRY_X86_INLINE_HPP
 #define CPU_X86_CONTINUATIONENTRY_X86_INLINE_HPP
 
-#include "runtime/continuationEntry.hpp"
-
 #include "oops/method.inline.hpp"
+#include "runtime/continuationEntry.hpp"
```

Comment:
I believe this is the "corresponding .hpp" in "All .inline.hpp files should include their corresponding .hpp file as the first include line..." from the style-guide, so should be kept at the top.